docs/server-config.md

```diff
@@ -125,6 +127,10 @@ settings._
 # Default value is 100 Mb.
 # TABPY_MAX_REQUEST_SIZE_MB = 100
 
+# Toggle for evaluate API
```

Comment:
You probably want to update https://github.com/tableau/TabPy/blob/master/tabpy/tabpy_server/common/default.conf with the same change.